src/main/scala/coupledL2/tl2chi/MSHR.scala

```diff
@@ -173,6 +173,7 @@ class MSHR(implicit p: Parameters) extends TL2CHIL2Module with HasCHIOpcodes {
 
   val hitDirty = dirResult.hit && meta.dirty || probeDirty
   val hitDirtyOrWriteBack = hitDirty || req.snpHitRelease && req.snpHitReleaseWithData
+  val snpNested = req.snpHitRelease && req.snpHitReleaseWithData
```

Comment:
Suggested better enchantment signals naming here:
```suggestion
  val hitWriteBack = req.snpHitRelease && req.snpHitReleaseWithData
  val hitDirtyOrWriteBack = hitDirty || hitWriteBack
```

Reply:
fixed in 9c8d6f2feabedae70f9310582b27c24e98ee3f69